aie_runtime_lib/CMakeLists.txt

```diff
@@ -34,7 +34,9 @@ function(add_aie_runtime_libs arch)
   set(INSTALLS
       chess_intrinsic_wrapper.cpp
       lut_based_ops.cpp
-      lut_based_ops.h)
+      lut_based_ops.h
+      tanh.h
+      tanh.cpp)
```

Comment:
Why are we copying these (.cpp) to the install directory? Aren't we using liblut_based_ops.a for that?